src/services/todo.js

```diff
@@ -0,0 +1,20 @@
+import { checkError, client } from './client';
+export async function fetchTodos() {
```

Comment:
adding some ordering here will make it so the todos don't shuffle when they are completed.